package/sap-installation-wizard.spec

```diff
@@ -20,27 +20,24 @@ Name:           sap-installation-wizard
 Summary:        Installation wizard for SAP applications
 License:        GPL-2.0+
 Group:          System/YaST
-Version:        4.5.1
+Version:        4.5.2
 Release:        0
-PreReq:         /bin/mkdir %insserv_prereq %fillup_prereq yast2
+PreReq:         /bin/mkdir %fillup_prereq yast2
 BuildRequires:  yast2
-Requires:       HANA-Firewall
 Requires:       autoyast2
 Requires:       autoyast2-installation
 Requires:       rubygem(%{rb_default_ruby_abi}:nokogiri)
 Requires:     	xfsprogs
-%if ! %{defined _SAPBOne}
-Requires:       sap-netscape-link
-Requires:       saprouter-systemd
-Requires:       yast2-hana-firewall
-Requires:       yast2-sap-scp
-Requires:       yast2-sap-scp-prodlist
-Requires:       saptune
-%endif
+Suggests:       HANA-Firewall
```

Comment:
The question is how much are these packages needed.

The suggested packages are not installed by default, maybe it should be `Recommends` instead. 

Recommended packages are installed by default if they are available and do not break other dependencies.

Reply:
I've changed Recommends for HANA-Firewall and saptune.